app/client/platforms/openai.ts

```diff
@@ -181,17 +182,19 @@ export class ChatGPTApi implements LLMApi {
         const content = visionModel
           ? await preProcessImageContent(v.content)
           : getMessageTextContent(v);
-        messages.push({ role: v.role, content });
+        if(!(isO1 && v.role === "system"))
+          messages.push({ role: v.role, content });
       }
 
+      // O1 not support image, tools (plugin in ChatGPTNextWeb) and system, stream, logprobs, temperature, top_p, n, presence_penalty, frequency_penalty yet.
```

Comment:
`shouldStream` will depend on `isO1` (see line: 210)

Reply:
Sorry, it seems there were some merge issues when migrating the code from my private repository. I will add it later.

Reply:
fix it in d0dce654bffead1971600a5feb313d9079800254